### DISCUSSION TITLE
[2933700] Hades II Technical Test {本体}

### DISCUSSION THREAD

**Discussion by Ddfche** [pjy612/SteamManifestCache#616] [Share]
### Steam 游戏ID

2933700

### 密钥列表

```hash
2933701;9d50bb8e55a8559ca5653d0a86ccf171cda8972323a5e7ec260ee0621c6dfa8b
2933703;124d209bb15dc686ce48f8dbc0a1ccb54f96a95cce319e99a0258e9765c23ef9
```

### 清单附件

[2933700.zip](https://github.com/pjy612/SteamManifestCache/files/15094719/2933700.zip)

### 备注

code:A4FCB120DDCF5B2F
email:xingf605@163.com

**Comment by pjy612** [comment]
beta 测试 很难评啊... 自取所需吧...

**Reply by ZEERDEER** [reply]
~刚好我忘了申请，现在申请满了，说是审核，估计不会给资格了✓~

原来 Playtest 是无法体验的
